sdk/python/feast/job_service.py

```diff
@@ -62,9 +68,15 @@ def _job_to_proto(self, spark_job: SparkJob) -> JobProto:
 
     def StartOfflineToOnlineIngestionJob(self, request, context):
         """Start job to ingest data from offline store into online store"""
-        context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-        context.set_details("Method not implemented!")
-        raise NotImplementedError("Method not implemented!")
+        feature_table = self.client.get_feature_table(
+            request.table_name, request.project
+        )
+        job = self.client.start_offline_to_online_ingestion(
```

Comment:
can we use `launcher.start_offline_to_online` instead of client here?

Reply:
the reason is: client may have different (user-oriented) API (like accepting strings instead of feature table, dataframes instead of source) and it's not really idempotent (and being already called once)

Reply:
Done in the fork: https://github.com/feast-dev/feast/pull/1129